File: Python/minMax.py
def findMinFiveSum(arr):
    sumMinFive = 0
    for i in range(4):
        sumMinFive = sumMinFive + arr[i]
    return sumMinFive    

def findMaxFiveSum(arr):
    sumMaxFive = 0
    for i in range(4):
        sumMaxFive = sumMaxFive + arr[len(arr)-i-1]
    return sumMaxFive    
           
def miniMaxSum(arr):
    arr.sort()
    print(findMinFiveSum(arr), findMaxFiveSum(arr))
    # Write your code here

File: JS-Typescript/MinMax.js
function findMinSum(arr){
    let sum=0;
    for(let i = 0; i < 4; i++){
        sum = sum + arr[i]; 
    }
    return sum;
}

function findMaxSum(arr){
    let sumMax=0;
    for(let i = 0; i < 4; i++){
        sumMax = sumMax + arr[arr.length-i-1]; 
    }
    return sumMax;
}

function miniMaxSum(arr) {
    // Write your code here
    arr.sort();
    console.log(findMinSum(arr)+" "+findMaxSum(arr))
    
}